test/unit_tests.vim

```diff
@@ -445,6 +445,25 @@ def Test_LspRename()
     }
   END
   assert_equal(expected, getline(1, '$'))
+
+  cursor(1, 1)
+  search('counter')
+  LspRename countvar
+  var expected2: list<string> =<< trim END
+    void F1(int countvar)
+    {
+	countvar = 20;
+
+	++countvar;
+    }
+
+    void F2(int count)
+    {
+	count = 5;
+    }
+  END
+  assert_equal(expected2, getline(1, '$'))
+  sleep 100m
```

Comment:
I think the tests are flaky, as a new `textDocument.publishDiagnostics` will be received after a rename. Causing the test `Test_LspDiagsUpdated_Autocmd` to pick up one to many autocmd triggers. The 100ms delay seems to delay the execution enough for the ls to be able to send `textDocument.publishDiagnostics` before continuing onto the tests in `Test_LspDiagsUpdated_Autocmd`